plover/assets/dict_unsorted.json

```diff
@@ -121866,7 +121866,6 @@
 "KWRA/AUL": "y'all",
 "KWRAOU/AUL": "y'all",
 "KWRAB/PWA/TKAB/PWA": "yabba dabba",
-"KWRAT": "yacht",
```

Comment:
Please don't delete those - that's exactly how I'd try to write "yacht" so it's a perfectly valid entry. And who doesn't like the Flintstones?